spring-cloud-dataflow-server/pom.xml

```diff
@@ -53,7 +53,12 @@
 			<artifactId>spring-cloud-dataflow-rest-client</artifactId>
 			<scope>test</scope>
 		</dependency>
-
+		<dependency>
+			<groupId>net.javacrumbs.json-unit</groupId>
+			<artifactId>json-unit</artifactId>
+			<version>1.25.0</version>
```

Comment:
No version in child poms

Reply:
right, will move it to s-c-dataflow-dependeices